setup.py

```diff
@@ -4,7 +4,8 @@
 from os.path import exists
 from setuptools import setup
 
-setup(name='dask',
```

Comment:
not naming the package dask would be a good start :P

Reply:
I don't even know why I put that there?!